src/grisp_connect_api.erl

```diff
@@ -134,6 +205,8 @@ error_atom(-3)  -> device_already_linked;
 error_atom(-4)  -> invalid_token;
 error_atom(-10) -> grisp_updater_unavailable;
 error_atom(-11) -> already_updating;
+error_atom(-12) -> boot_system_not_validated;
+error_atom(-13) -> validate_from_unbooted;
```

Comment:
This function is only used in `handle_response`, so it only handles error codes coming from grisp_manager.
For this reason errors` 10 11 12 13` are not needed.

We have these same lines in grisp_manager https://github.com/stritzinger/grisp_manager/blob/51e1be1b137e14534111cd4535df80495cf4a9ab/src/grisp_manager_device_api.erl#L201

Reply:
Oki, I changed it :)